### PR TITLE
Ecto: From chapter 4 to 5

### PR DESCRIPTION
**Pull request by Vretamoza**

### CODE REVIEW

**Comment by juanhurtado10** [comment]
@Vretamoza A rebase is required here :)

File: friends/lib/friends/person.ex [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  import Ecto.Changeset
+
+  schema "people" do
+    field :name, :string
+    field :age, :integer, default: 0
+  end
+
+  def changeset(struct, params) do
+    struct
+    |> cast(params, [:name, :age])
```
Comment (juanhurtado10) [inline friends/lib/friends/person.ex:12]: @Vretamoza what does `cast/3` do?

File: friends/lib/friends/person.ex [hunk 2]
```diff
@@ -0,0 +1,32 @@
+defmodule Friends.Person do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "people" do
+    field :name, :string
+    field :age, :integer, default: 0
+  end
+
+  def changeset(struct, params) do
```
Comment (juanhurtado10) [inline friends/lib/friends/person.ex:10]: @Vretamoza what is a changeset?

**Review by juanhurtado10 — CHANGES_REQUESTED** [review]

File: friends/lib/friends/person.ex (hunk 1 above)
Comment (Vretamoza) [reply friends/lib/friends/person.ex:12]: The cast function creates a changeset. Unlike the function change, it accepts three parameters, where the first one is the original data, the second is the changes we want to make, and the third is a list of fields that we allow to be changed.

File: friends/lib/friends/person.ex (hunk 2 above)
Comment (Vretamoza) [reply friends/lib/friends/person.ex:10]: A changeset is a struct that saves some changes and some validation rules that allows us to save data in the database just if it is valid.

**Pull request merged**